scripts/artifactory-publish.sh

```diff
@@ -5,4 +5,18 @@ echo 'Deploying navi webservice models to artifactory'
 
 cd packages/webservice
 
-./gradlew -p models artifactoryPublish
+BRANCH=$(git branch --show-current);
+
+echo "Deploying navi webservice models to artifactory for branch $BRANCH"
+
+if [ $BRANCH = 'master' ]
+then
+    echo 'Publishing beta build...'
+    ./gradlew -PpublishTag=beta -p models artifactoryPublish
+elif [ $BRANCH = '0.2.x-alpha' ]
+then
+    echo 'Publishing alpha build...'
+    ./gradlew -p models artifactoryPublish
+else
+    echo 'Not a publishable branch'
```

Comment:
Should we fail in this case?

```suggestion
    echo 'Not a publishable branch'
    exit 1;
```

Reply:
Nah I don't think it's necessary to fail the build if someone runs the workflow from an odd branch.